Marlin/src/gcode/bedlevel/abl/G29.cpp

```diff
@@ -399,6 +397,10 @@ G29_TYPE GcodeSuite::G29() {
       points[0].z = points[1].z = points[2].z = 0;  // Probe at 3 arbitrary points
     #endif
 
+    #if BOTH(AUTO_BED_LEVELING_BILINEAR, EXTENSIBLE_UI)
+      ExtUI::onMeshLevelingStart();
+    #endif
```

Comment:
Is this only used for Bilinear? Not MBL or UBL? Previously it looks like it was always enabled, regardless of leveling.

Reply:
MBL and UBL should have it in their respective files, this file only handles 3 point, linear, and bilinear.

Reply:
Oh, of course. I didn't look at which file I was in.

Reply:
> 
> 
> Is this only used for Bilinear? Not MBL or UBL? Previously it looks like it was always enabled, regardless of leveling.

Shouldn't that statement apply that to the PR of strain gauge support as well?

Reply:
Thats in the other PR, and everything for that is implemented directly in the probe class so it should not care what leveling routine is calling it.